### DISCUSSION TITLE
Introducing RamanFiber from XLSX

### DISCUSSION THREAD

**Discussion by gv100204** [Telecominfraproject/oopt-gnpy#443] [Q&A] [answered]
Hi all,

I would like to know if it's possible to introduce RamanFiber directly from XLSX or if the only way is to add manually to the Json.

Thanks for your help!

**Comment by jktjkt** [comment, marked as answer]
Hi, the XLS frontend does not support that.

**Reply by gv100204** [reply]
Thank you! So the only way to insert a RamanFiber is on the JSON file, correct?

**Reply by jktjkt** [reply]
Yes. Note that you can have GNPy save the topology as a JSON file (see the `--help` output) and adjust that file via something like [`jq`](https://stedolan.github.io/jq/).
